core/src/main/java/org/pentaho/platform/dataaccess/datasource/api/AnalysisService.java

```diff
@@ -119,9 +120,12 @@ public List<String> getAnalysisDatasourceIds() {
     Set<String> ids = metadataDomainRepository.getDomainIds();
     for ( MondrianCatalog mondrianCatalog : mockMondrianCatalogList ) {
       String domainId = mondrianCatalog.getName() + METADATA_EXT;
-      if ( !ids.contains( domainId ) || !isDSWDatasource( domainId ) ) {
+      String createdWithProperty = mondrianCatalog.getConnectProperties().get( "CreatedWith" );
+      if ( ( !ids.contains( domainId ) || !isDSWDatasource( domainId ) ) &&
+        ( createdWithProperty == null || !createdWithProperty.equalsIgnoreCase( "SME" ) ) ) {
```

Comment:
Also refactor the logic into a function. Something like `boolean shouldListCatalog( MondrianCatalog catalog )`